pages/myinfo/account-delete/index.page.tsx

```diff
@@ -137,18 +161,28 @@ export default function AccountDelete() {
     </>
   );
 
+  const accountDeleteTitle = isMobile
```

Comment:
요부분이 중첩 삼항연산자라 가독성이 떨어져서 아래와 같이 괄호를 넣는건 어떨까요 ? : ) 
```suggestion
const accountDeleteTitle = isMobile
  ? (step === 'step1' ? 'flex-col mt-auto' : 'mr-auto')
  : 'justify-between';
```